tutormfe/plugin.py

```diff
@@ -90,6 +89,14 @@ def get_mfes() -> dict[str, MFE_ATTRS_TYPE]:
     return MFE_APPS.apply({})
 
 
+@functools.lru_cache(maxsize=None)
```

Comment:
Make sure to use [tutor.hooks.lru_cache](https://docs.tutor.edly.io/reference/api/hooks/index.html#tutor.hooks.lru_cache) (and fix get_mfes above)

Reply:
Got it, will do!